src/components/Table.tsx

```diff
@@ -0,0 +1,31 @@
+type TableArray = {
+  rows: rows[];
+  label: string;
+};
+
+type rows = {
+  key: number;
+  name: string;
+  email: string;
+};
+
+export default function Table({ rows, label }: TableArray) {
```

Comment:
[IMO]
これだと、何のlabelかわからないので、 nameLabel とかにしても良いかなと思いました。

```suggestion
export default function Table({ rows, nameLabel }: TableArray) {
```